core/vm/interpreter.go

```diff
@@ -34,8 +36,18 @@ var (
 )
 
 const (
+	// These are keys for the interruptCtx
 	InterruptCtxDelayKey       = "delay"
 	InterruptCtxOpcodeDelayKey = "opcodeDelay"
+
+	// InterruptedTxCacheSize is size of lru cache for interrupted txs
+	InterruptedTxCacheSize = 90000
+
+	// InterruptedTxCacheKey gets you the pointer to lru cache storing the interrupted txs
+	InterruptedTxCacheKey = "interruptedTxCache"
+
+	// InterruptedTxContext_currenttxKey gets you the hash of the current tx being executed
+	InterruptedTxContext_currenttxKey = "currenttx"
```

Comment:
Could you make context key private and rewrire the code in a way like https://go.dev/play/p/2YNOSVTioB8